packages/manager/modules/navbar/src/notifications-menu/controller.js

```diff
@@ -67,35 +85,16 @@ export default class NotificationsCtrl {
     )
       .then((notifications) =>
         notifications.map((notification) =>
-          this.NavbarNotifications.convertToSubLink(notification),
+          this.NavbarNotifications.constructor.convertToSubLink(notification),
         ),
       )
       .catch(() => undefined);
   }
 
-  acknowledgeAll() {
+  openMenu() {
```

Comment:
Can the directives be used here ? to avoid adding a function for only tracking purpose